README.md

```diff
@@ -12,7 +12,24 @@ Why is this needed: when images are in S3, Amazon EC2 instance which runs MediaW
 
 3\) Run `composer install` from `/var/www/html/w/extensions/AWS` (to download dependencies). If you don't have Composer installed, see https://www.mediawiki.org/wiki/Composer for how to install it.
 
-4\) Create an S3 bucket for images, e.g. `wonderfulbali234`. Note: this name will be seen in URL of images.
+4\) Create an S3 bucket for images, e.g. `wonderfulbali234`. Note: this name will be seen in URL of images. The bucket should have a policy that enables its contents to be public readable by default. For this reason, you should use a unique bucket for your Wiki and not reuse an existing one. For example, this policy will make `wonderfulbali234` accessible by default to the public:
+
+```
+{
```

Comment:
This policy is not needed.
We set ACL for each newly uploaded file in PutObject call:
https://github.com/edwardspec/mediawiki-aws-s3/blob/bcdabbbb861eeac758fb3de1f7f1391777b7eb6d/s3/AmazonS3FileBackend.php#L287

Furthermore, even on the public wiki, not all objects in the bucket should be public.
For example, deleted images are not public, because they could have been deleted due to copyright violation or something.